src/Raven.Client/Documents/Operations/CollectionStatistics.cs

```diff
@@ -1,17 +1,70 @@
 ﻿using System.Collections.Generic;
+using Raven.Client.Util;
+using Sparrow.Json.Parsing;
 
 namespace Raven.Client.Documents.Operations
 {
-    public class CollectionStatistics
+    public class CollectionStatistics : IDynamicJson
     {
         public CollectionStatistics()
         {
             Collections = new Dictionary<string, long>();
         }
 
-        public int CountOfDocuments { get; set; }
-        public int CountOfConflicts { get; set; }
+        public long CountOfDocuments { get; set; }
+        public long CountOfConflicts { get; set; }
```

Comment:
@ppekrol Backward compact issue here?

Reply:
Yes, should be 'int'. We can change that in 5.0 only.